test/JDBC/expected/objectproperty.out

```diff
@@ -1069,12 +1589,76 @@ int
 ~~END~~
 
 
+-- tsql user=objectproperty_login_1 password=12345678
+use db1
+go
+
+-- user does not have permission on the object (should return NULL)
+SELECT OBJECTPROPERTY(OBJECT_ID('IsTrigger_trigger', 'TR'), 'IsTrigger')
+GO
+~~START~~
+int
+<NULL>
+~~END~~
+
+
+-- user does not have permission on the object (should return NULL)
+SELECT OBJECTPROPERTY(OBJECT_ID('IsTrigger_table', 'TR'), 'IsTrigger')
+GO
+~~START~~
+int
+<NULL>
+~~END~~
+
+
+-- tsql
+use db1
+go
+
 -- Cleanup
 DROP TABLE IsTrigger_table
 GO
 
+
+-- logical schema is 'information_schema', we should return NULL
+select objectproperty(object_id('information_schema.columns'), 'IsView')
+go 
+use master
+go
+~~START~~
+int
+<NULL>
+~~END~~
```

Comment:
Seems like you have not validated the behaviour:
```
1> select objectproperty(object_id('information_schema.columns'), 'IsView')
2> go
           
-----------
          1

(1 rows affected)

Reply:
Based on this, we will need to rewrite information_schema name to information_schema_tsql